.github/workflows/ci.yml

```diff
@@ -29,38 +29,40 @@ jobs:
       fail-fast: false
       matrix:
         env:
-          - python: 3.6
-            TOXENV: py36-django22-djangorestframework310
           - python: 3.7
             TOXENV: py37-django22-djangorestframework310
           - python: 3.8
             TOXENV: py38-django22-djangorestframework310
           - python: 3.9
             TOXENV: py39-django22-djangorestframework310
-          - python: 3.6
-            TOXENV: py36-django22-djangorestframework312
           - python: 3.7
             TOXENV: py37-django22-djangorestframework312
           - python: 3.8
             TOXENV: py38-django22-djangorestframework312
           - python: 3.9
             TOXENV: py39-django22-djangorestframework312
-          - python: 3.6
-            TOXENV: py36-django30-djangorestframework312
           - python: 3.7
             TOXENV: py37-django30-djangorestframework312
           - python: 3.8
             TOXENV: py38-django30-djangorestframework312
           - python: 3.9
             TOXENV: py39-django30-djangorestframework312
-          - python: 3.6
-            TOXENV: py36-django31-djangorestframework312
```

Comment:
Since this library is widely used, I think we can leave these old builds (this one and the one above) to maintain wide support to legacy versions.